va/va.go

```diff
@@ -258,7 +271,8 @@ func (d *http01Dialer) Dial(_, _ string) (net.Conn, error) {
 	address := net.JoinHostPort(v4[0].String(), d.record.Port)
 	d.record.AddressUsed = v4[0]
 	realDialer = d.realDialer()
-	return realDialer.Dial("tcp", address)
+	conn, err := realDialer.DialContext(ctx, "tcp", address)
```

Comment:
nit: was there a reason to capture the two returns into `conn, err` and then `return` them explicitly vs changing the existing `return realDialer.Dial("tcp", address)` into `return realDialer.DialContext(ctx, "tcp", address)`? I think the latter is a little cleaner (smaller diff anyway :-))